### PR TITLE
(FM-8230) Convert testing to litmus

### PR DESCRIPTION
**Pull request by eimlav**

### CODE REVIEW

File: spec/acceptance/has_interface_with_spec.rb [hunk 1]
```diff
@@ -1,52 +0,0 @@
-require 'spec_helper_acceptance'
```
Comment (tphoney) [inline spec/acceptance/has_interface_with_spec.rb]: where did the acceptance tests go  ?

File: spec/functions/loadjson_spec.rb [hunk 2]
```diff
@@ -134,6 +136,7 @@
       let(:basic_auth) { { :http_basic_authentication => ['', ''] } }
 
       it {
+        # require 'pry'; binding.pry
```
Comment (RandomNoun7) [inline spec/functions/loadjson_spec.rb]: Should probably remove?

File: spec/acceptance/build_csv.rb [hunk 3]
```diff
@@ -1,89 +0,0 @@
-# vim: set sw=2 sts=2 et tw=80 :
```
[the remark this answers is not in the document]
Comment (eimlav) [reply spec/acceptance/build_csv.rb]: It contained helper functions that are no longer used anywhere in the test suite so I removed it.

File: spec/acceptance/has_interface_with_spec.rb (hunk 1 above)
Comment (eimlav) [reply spec/acceptance/has_interface_with_spec.rb]: The acceptance test duplicated functionality tested in unit. Do you think this is worth keeping as an acceptance test?

File: spec/functions/loadjson_spec.rb (hunk 2 above)
Comment (eimlav) [reply spec/functions/loadjson_spec.rb]: Yup!

**Comment by eimlav** [comment]
@tphoney I have left some updates on your comments.

**Comment by eimlav** [comment]
@RandomNoun7 Removed pry. If everything else looks good would you be able to merge? Cheers!

File: spec/acceptance/anchor_spec.rb [hunk 4]
```diff
@@ -15,11 +15,12 @@ class anchorrefresh {
...
       include anchorrefresh
-    DOC
-    it 'effects proper chaining of resources' do
-      apply_manifest(pp, :catch_failures => true) do |r|
-        expect(r.stdout).to match(%r{Anchor\[final\]: Triggered 'refresh'})
-      end
+    MANIFEST
+  end
+
+  it 'applies manifest' do
```
Comment (RandomNoun7) [inline spec/acceptance/anchor_spec.rb]: This `it` description is going to be difficult to understand in CI results. There will be no indication of what this test is trying to do. Maybe something a little more descriptive up in the describe string?

Comment (eimlav) [reply spec/acceptance/anchor_spec.rb]: @RandomNoun7 updated to be a more explicit

**Pull request merged**